docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  web:
+    ports:
+     - '8080:8080'
+    build:
+      context: ./
```

Comment:
Could you add a `volume` pointing to copy changes done to the code automatically?